src/main/java/DilemmaDetector/Simulator/Vector2.java

```diff
@@ -25,6 +25,13 @@ public Vector2 add(Vector2 other){
         return this;
     }
 
+    public Vector2 sub(Vector2 other){
```

Comment:
wystarczy dodać ujemną wartość, można nawet tą metodę tak zmienić, ale to jak chcesz :P